roles/st2repos/tasks/redhat.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Install RPM key for StackStorm
+  become: yes
+  rpm_key:
+    state: present
+    key: https://packagecloud.io/StackStorm/stable/gpgkey
+
+- name: Add ST2 Repo
+  become: yes
+  yum_repository:
+    name: stackstorm
+    description: Stackstorm Repo
+    file: st2
```

Comment:
I'm for hardcoding safe values we can use easily via `yum_repository`, without auto-generating them.